### DISCUSSION TITLE
.Net 8

### DISCUSSION THREAD

**Discussion by KryptoBeard** [AutoMapper/AutoMapper.Extensions.OData#202] [General]
Does anyone see any potential issues with using this with .net 8 and EfCore 8? Going to attempt to upgrade from 7, but I have this package as a dependency throughout my Api.

Thanks!

**Comment by BlaiseD** [comment]
You'll probably find issues.  You can create a PR similar to [this one](https://github.com/AutoMapper/AutoMapper.Extensions.OData/pull/187).  In this case only the  ".yml" and ".csproj" files will need changing.
